parser/src/main/java/io/specmesh/apiparser/model/Message.java

```diff
@@ -40,12 +42,17 @@
 @SuppressFBWarnings
 @SuppressWarnings({"unchecked", "rawtypes"})
 public class Message {
+    static final Set<String> SUPPORTED_TYPES =
```

Comment:
Feels like a strange place to define this. But if you do want to define it here, considering naming it better.  This is supported _Kafka_ types.

Reply:
hmm - ill move it to the main API - this package is just the parser, not kafka even though we are 'borrowing' them from kafka